tests/flow/Install_RedisGears.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ROOT=$(cd $HERE/../..; pwd)
```

Comment:
ROOT=$(cd $HERE/../../ && pwd)

By combining them with a semi-colon (;) instead of double-ampersand (&&) your command will succeed, even if the directory fails to exist. This could have unintended consequences in many cases.